README.md

```diff
@@ -0,0 +1,11 @@
+# Omega robot
+--*In developpement*--
+
+This is a bot for our discord, it will include many features. 
```

Comment:
According to me, "it will include many features" is not a very useful information in a README.